webrtc.html

```diff
@@ -8055,17 +8055,20 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   the first argument.</p>
                 </li>
                 <li>
-                  <p>If the second (dictionary) argument is present, set
-                  <var>channel</var>'s <code><a data-for=
+                  <p>Let <var>init</var> be the second (dictionary) argument,
```

Comment:
Filed https://github.com/heycam/webidl/issues/349 about this bit. In the meantime, I suggest writing prose that just assumes that the dictionary is there. https://wicg.github.io/mediasession/#dom-mediametadata-mediametadata is an example of this.